avionics/common/include/states/landed.h

```diff
@@ -30,6 +30,8 @@ class Landed : public IState {
         constexpr static unsigned int LANDED_POLLING_INTERVAL = 5000;
         Scheduler::get_task(static_cast<int>(TaskID::ReadEvalLog)).period =
             Hal::ms(LANDED_POLLING_INTERVAL);
+
+        Scheduler::unschedule(static_cast<int>(TaskID::RestartCamera));
         cam_.stop_record();
```

Comment:
Might want to put a comment here noting that  stop_record ensures that the camera stops recording, since the state of the RestartCamera task is unknown at this point